src/Resources/app/administration/src/module/sw-extension-store/component/sw-extension-store-statistics-promotion/index.js

```diff
@@ -0,0 +1,61 @@
+import template from './sw-extension-store-statistics-promotion.html.twig';
+import './sw-extension-store-statistics-promotion.scss';
+
+const STATISTICS_APP_NAME = 'StatisticsService';
+
+export default Shopware.Component.wrapComponentConfig({
```

Comment:
This is basically the same component as on the dashboard isn't it? (just with some different snippets) Maybe we can reuse the code instead of copying it.

Reply:
I tried to solve it with vue composables but it couldn't be done like that because [we don't have support for the composition API in the admin yet](https://shopware-ag.slack.com/archives/C05A777AF8V/p1713516848451519?thread_ts=1713515396.784539&cid=C05A777AF8V)